frontend/src/routes/forget/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+    import '$styles/Auth.pcss';
+    import { enhance } from '$app/forms';
+</script>
+
+<html class="h-full">
```

Comment:
Could you remove the <html> tag? There is some issue including it with svelte.
<img width="685" alt="Screenshot 2024-03-14 at 1 35 09 PM" src="https://github.com/cics-web-dev/Chem-Web-App/assets/103079472/e2b3a05a-0f5d-4de8-b05e-253d67a6f4f6">